### PR TITLE
Style.css: Added support for a vertical bar

### PR DESCRIPTION
**Pull request by jacob-and-nathan**
A vertical bar is an element that was used quite a lot in windows 98. This code makes it easy to do so...

### CODE REVIEW

**Comment by jdan** [comment]
Hey! Wanna drop a screenshot?

**Comment by jacob-and-nathan** [comment]
Yep:
Screenshot 2020-04-26 at 10 05 44 PM

This is an example of how I used this element in a separate, personal project.

File: style.css [hunk 1]
```diff
@@ -129,6 +129,14 @@ button {
   padding: 0 12px;
 }
 
+.vertical-bar{
+  width: 4px;
+  height: 20px;
+  background: #c0c0c0;
+  box-shadow: inset -1px -1px var(--window-frame), inset 1px 1px var(--button-highlight), inset -1px -1px var(--button-highlight), inset 1px 1px var(--window-frame);
+  border-radius: 5px;
```
Comment (jdan) [inline style.css:137]: image

Do we want a border radius?

File: style.css [hunk 2]
```diff
@@ -129,6 +129,14 @@ button {
   padding: 0 12px;
 }
 
+.vertical-bar{
+  width: 4px;
+  height: 20px;
+  background: #c0c0c0;
+  box-shadow: inset -1px -1px var(--window-frame), inset 1px 1px var(--button-highlight), inset -1px -1px var(--button-highlight), inset 1px 1px var(--window-frame);
```
Comment (jdan) [inline style.css:136]: Can we build this out of `var(--border-raised-outer), var(--border-raised-inner);` or something similar?

File: style.css (hunk 1 above)
Comment (jacob-and-nathan) [reply style.css:137]: Actually, I think you're right - the border-radius can be removed.

File: style.css (hunk 2 above)
Comment (jacob-and-nathan) [reply style.css:136]: Yes, that would make more sense. I haven't done very much with CSS variables before, so what's the syntax?

Comment (jdan) [reply style.css:136]: Give the box-shadow for `button` a peek (a few lines above)

**Comment by jacob-and-nathan** [comment]
(I have updated the file & pull request)

**Comment by jacob-and-nathan** [comment]
Yes, I have changed both those things and updated the pull request.

**Comment by jdan** [comment]
Great work, thank you!

I'll merge this as-is and we'll see if it gets any usage :) Is this element used solely in menu bars? If we ever build one of those we'll have to add it the docs.

**Comment by jacob-and-nathan** [comment]
Looking at various images, it looks like this element is also used on menus in programs like internet explorer (see image), to separate out different sections of the design
Screenshot 2020-04-27 at 4 47 03 PM

**Comment by veleek** [comment]
@jdan - for draggable/organizable menus this bar is used as a drag handle.

Here's the equivalent in the current version of VS:
image

**Pull request merged**